exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/DatabaseConfig.kt

```diff
@@ -43,6 +45,18 @@ class DatabaseConfig private constructor(
          * Default attempts are 3
          */
         var defaultRepetitionAttempts: Int = 3,
+        /**
+         * The minimum time to wait before retrying a transaction if SQLException happens
+         * Can be overridden on per-transaction level by specifying `minRepetitionDelay` parameter on call
+         * Default minimum delay is 0
+         */
+        var defaultMinRepetitionDelay: Long = 0,
+        /**
+         * The maximum time to wait before retrying a transaction if SQLException happens
+         * Can be overridden on per-transaction level by specifying `maxRepetitionDelay` parameter on call
+         * Default maximum delay is 0
+         */
+        var defaultMaxRepetitionDelay: Long = 0,
```

Comment:
Please edit the KDocs to use a specific amount of time, so that a `Long` value of milliseconds isn't mistakenly provided -> "The minimum seconds ..." | "The minimum number of seconds ...".

Or choose milliseconds as the accepted `Long` and adjust `Thread.sleep()` and the logged warning accordingly.

Reply:
I changed the accepted Long to milliseconds and updated the KDocs